site_v5.yml

```diff
@@ -43,67 +43,77 @@
         version: 1.0.0
         sha256sum: 5e5de66e3517d30ff19ef66cf7a4aa5443b861d83e36a75e85845b007a03afbf
       aaf_shib_ext:
-        baseurl: "{{ aaf_binaries.baseurl }}/jars/aaf_shib_ext"
-        version: 2.0.0
-        sha256sum: 5244b2c2565ed40fe66ee8d551ad5e5f1aac0c26c06eace521658792297d464c
+        baseurl: "{{ aaf_binaries.baseurl }}/jars/aaf-shibext"
+        version: 5.1.2
+        sha256sum: 14913ec24500f4b952773c36c72cdcdc905e1e0be0ce607cf6fbba86073336c1
       keystore:
         baseurl: "{{ aaf_binaries.baseurl }}/keystore"
         version: "Unknown"
         sha256sum: ae4185b2f0bb1af00abc6a4502fbfbdc6a90aec65c7bcee08e37a1bc20de5ac1 
-      logback_access:
-        baseurl: "{{ aaf_binaries.baseurl }}/jars/logback"
-        version: 1.2.10
-        sha256sum: 8462a7d983a10700c2cc559b7df4254f3a21c2611c32476cb5a6702ce51524c9
+      jcl_over_slf4j:
+        baseurl: "{{ aaf_binaries.baseurl }}/jars/slf4j"
+        version: 2.0.5
+        sha256sum: 95fa90eb540df07b6a2fa70f3613761180d07c9c8f1c299e3f9a5c943313a2d6
       logback_classic:
         baseurl: "{{ aaf_binaries.baseurl }}/jars/logback"
-        version: 1.2.10
-        sha256sum: 3160ae988af82c8bf3024ddbe034a82da98bb186fd09e76c50543c5b9da5cc5e
+        version: 1.3.6
```

Comment:
It seems that Jetty 11/12 uses logback-classic 1.5.6

https://mvnrepository.com/artifact/ch.qos.logback/logback-classic/1.5.6 (Released on Apr 17, 2024)
https://repo1.maven.org/maven2/ch/qos/logback/logback-classic/1.5.6/logback-classic-1.5.6.jar

Reply:
Have added a new issue to upgrade to latest version.